Mathlib/Topology/Algebra/InfiniteSum/Basic.lean

```diff
@@ -66,7 +66,10 @@ def Summable (f : β → α) : Prop :=
 
 /-- `∑' i, f i` is the sum of `f` it exists, or 0 otherwise -/
 irreducible_def tsum {β} (f : β → α) :=
```

Comment:
I think it would be nice to explain this a bit, either in the docstring or simply in a comment, just in case people don't have non-Hausdorff topologies in mind while reading the definition.

Otherwise LGTM!

Reply:
Good idea. I've put it in a comment, because I think it's a little bit too specialized for people just wanting to use infinite sums and looking at the docstring.

Reply:
I agree that it's better in a comment.